$NetBSD: patch-configure,v 1.1 2015/11/07 22:56:13 he Exp $

Use $host_alias in the gcc-libdir path so it matches ${MACHINE_GNU_PLATFORM}
Treat NetBSD/powerpc as Linux powerpc.

--- configure.orig	2010-04-22 04:07:13.000000000 +0000
+++ configure
@@ -4647,7 +4647,7 @@ _ACEOF
    { $as_echo "$as_me:${as_lineno-$LINENO}: result: $gcc_version" >&5
 $as_echo "$gcc_version" >&6; }
 
-   DEST_LIBDIR=$prefix/lib/gcc-lib/$host/$gcc_version
+   DEST_LIBDIR=$prefix/lib/gcc-lib/$host_alias/$gcc_version
    SAVE_DIR=`pwd`
    PREFIX=$prefix
 
@@ -4911,6 +4911,10 @@ $as_echo "#define FPU_PPC_LINUX 1" >>con
 
      ;;
 
+     *netbsd*)
+$as_echo "#define FPU_PPC_LINUX 1" >>confdefs.h
+     ;;
+
      *)
        echo host = $host
        as_fn_error "Unknown powerpc architecture" "$LINENO" 5
